fdb-relational-core/src/main/java/com/apple/foundationdb/relational/api/ProtobufDataBuilder.java

```diff
@@ -71,7 +72,7 @@ public String getFieldType(String fieldName) throws SQLException {
     public boolean isPrimitive(int fieldNumber) throws SQLException {
         final var field = typeDescriptor.getFields().get(fieldNumber - 1);
         if (field == null) {
-            throw new RelationalException(String.format("Field with number <%d> does not exist", fieldNumber), ErrorCode.INVALID_PARAMETER).toSqlException();
+            throw new RelationalException(String.format(Locale.ROOT, "Field with number <%d> does not exist", fieldNumber), ErrorCode.INVALID_PARAMETER).toSqlException();
```

Comment:
We should update the release notes with this change, as it will be user visible for anyone with an alternate default locale. 
Something broad, like "The relational project is now specifying `Locale.ROOT` in all `String.format` calls, which may change number formatting if the default locale does not align`

Reply:
Done.